.github/actions/build/action.yaml

```diff
@@ -41,6 +41,11 @@ runs:
           cd src
           gn args --list --short --overrides-only out/${{ matrix.platform }}_${{ matrix.config }}
         shell: bash
+       - name: gn check
```

Comment:
gn.py is already adding --no-check arg: https://source.corp.google.com/h/github/youtube/cobalt/+/main:cobalt/build/gn.py;l=153?q=gn.py&ss=h%2Fgithub%2Fyoutube%2Fcobalt%2F%2B%2Frefs%2Fheads%2Fmain 

you need to change that.

We can run gn check separately or via gn.py - currently it's been run via gn.py .